.github/scripts/get-component-release-notes.js

```diff
@@ -1,3 +1,37 @@
+
+function getComponentName(name){
+    switch(name){
```

Comment:
Mapping can be  simpler  than adding many cases
```
const components = new Map([
  ["Codeflare", "codeflare"],
  ["Ray", "ray"],
["Kueue","kueue"],
["data-science-pipelines-operator", "Datascience Pipelines"]
]);

return components.get(name);
```
It will return value based on key if found else return `undefined`

Reply:
The plan is to remove the this coupling and print the component names as is. So this switch will be removed.